packages-ts/gauntlet-terra-contracts/src/commands/tooling/query.ts

```diff
@@ -0,0 +1,40 @@
+import { assertions, logger } from '@chainlink/gauntlet-core/dist/utils'
+import { TerraCommand } from '@chainlink/gauntlet-terra'
+import { CATEGORIES } from '../../lib/constants'
+export default class QueryContractCode extends TerraCommand {
+  static description = 'Query deployed contracts'
+  static examples = [`yarn gauntlet query --network=[NETWORK] --msg='QUERY' [CONTRACT_ADDRESS]`]
```

Comment:
I think currently we namespace the command under the category:

```suggestion
  static examples = [`yarn gauntlet tooling:query --network=[NETWORK] --msg='QUERY' [CONTRACT_ADDRESS]`]
```

Reply:
I modeled after the 'upload' command. Using tooling:query causes an error:

ABI not found for contract tooling
Error: Abstract: Contract tooling not found